### ISSUE TITLE
Question about n_backward_frames_to_reproject

### ISSUE THREAD

**Issue by RaymondWang987** [nesl/radar-camera-fusion-depth#26]
Hi authors, thanks for sharing the great work.

I have one question about the `--n_backward_frames_to_reproject` for preprocessing the Nuscenes dataset. Why `n_backward_frames_to_reproject` is set to 80 for the training and validation set, while set to 24 for the testing set? (in setup_dataset_nuscenes.sh and setup_dataset_nuscenes_test.sh)

Looking forward to your reply. Thanks!

**Comment by thecyclone** [comment]
We are not using the reprojected ground truth during testing, so it does not matter what you set the value to.

https://github.com/nesl/radar-camera-fusion-depth/blob/1e2cd8f4af5909bf2242acc6fb8caf031ccb7363/bash/run_fusionnet_nuscenes_test.sh#L14
